storage/aptosdb/src/lib.rs

```diff
@@ -1000,6 +1000,39 @@ impl DbReader for AptosDB {
         })
     }
 
+    fn get_gas_prices(
```

Comment:
please don't add this as a DB API. expose get_transaction_iter() instead.

Reply:
The `TransactionIter` is a type that's hidden here in the DB and can't get exposed via the StorageInterface

Reply:
K. let's unblock you first.